src/manager/node.c

```diff
@@ -376,8 +376,6 @@ static int node_match_heartbeat(UNUSED sd_bus_message *m, UNUSED void *userdata,
                 return 0;
         }
 
-        hirte_log_debugf("Heartbeat received from %s", node_name);
```

Comment:
for debugging this could be useful though, no?

Reply:
It makes all other debugging readlly hard because everything drowns in this one line over and over.